x-pack/legacy/plugins/apm/public/components/shared/Links/url_helpers.ts

```diff
@@ -39,6 +39,7 @@ export type APMQueryParams = {
   rangeTo?: string;
   refreshPaused?: string | boolean;
   refreshInterval?: string | number;
+  filter?: string;
```

Comment:
`filter` is a bit too generic, and collides with the concept of "ui filters". Can you name it `searchTerm` or something like that?

Reply:
Btw. I was hoping we could avoid having to declare it here, and just keep it in the local state of the component. Is that not possible?

Reply:
It's possible but I'd have to lift it to the parent components, in order to apply the same filter in other UI like flyouts.